recipes/darker/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "darker" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3a840c2cedec7e4d13c26c450d94f31bd872375450a0c057b9276cc9c2dbad5c
+
+build:
+  # noarch: python
+  skip: True  # [win]
```

Comment:
What's the issue with Windows? Was it just that `test -f` didn't work?

Reply:
There is currently an installation issue on Windows https://github.com/akaihola/darker/pull/120 

At some point we might be able to make this package noarch, but due to this Windows issue we can't for now

Reply:
akaihola/darker#120 has been merged and included in later versions – has this skip been removed?

Reply:
Yes it has been removed, you can see it here that the package is now noarch https://github.com/conda-forge/darker-feedstock/blob/main/recipe/meta.yaml#L13